caikit/interfaces/nlp/data_model/classification.py

```diff
@@ -82,6 +89,9 @@ class TokenClassificationResult(DataObjectBase):
     token_count: Annotated[
         Optional[int], FieldNumber(7)
     ]  # Length of tokens in the text
+    warnings: Annotated[
+        Optional[InputWarnings], FieldNumber(8)
```

Comment:
Two separate concerns on this
- `warnings` leads me to believe there could be multiple warnings, but the `InputWarnings` data object seems to only allow one message. Should that be singular and a list allowed here?
- Currently none of the tasks expect or return this https://github.com/caikit/caikit/blob/main/caikit/interfaces/nlp/tasks.py, so what is the rationale on putting the warnings on individual `TokenClassificationResult`? i.e. what kind of warnings are we expecting to return on a token level basis rather than an overall results basis?

Reply:
1. Per SW,  The ideal warning in the event of the guardrails user input detection warning should look something like this: 
`warnings" : [  {
      "id": "generation_prevented",
       "message": "Unsuitable input detected. Please check the detected entities on your input and try again with the unsuitable input removed."  } ]`
      
So the idea could be either a list of InputWarnings if multiple warnings? 

2. You are right - this should only be in ClassifiedGeneratedTextResult (in which case, I don't think I'd have to worry about breaking the order for field numbers